Source/Core/DolphinQt2/Debugger/CodeViewWidget.cpp

```diff
@@ -143,11 +143,11 @@ void CodeViewWidget::Update()
       item->setData(Qt::UserRole, addr);
 
       if (color != 0xFFFFFF)
-        item->setBackground(QColor(color).darker(200));
```

Comment:
This change was made as a compromise to make the debugger usable when using darker themes.

So, there are multiple solutions to this:
a) Check for the font color, decide how / if the colors have to be modified
b) Darken them but by a smaller value (100?)
c) Hard code the colors for this widget.